src/config/ThemeProvider.tsx

```diff
@@ -54,7 +54,13 @@ export default class ThemeProvider extends React.Component<
 
   static getDerivedStateFromProps(props, state) {
     const { useDark } = props;
-    if (useDark !== state.useDark) {
+    const isTheme = (theme) => {
```

Comment:
This function still doesn't make sense to me. Are we just checking `theme !== {}`. That seems a bit easier to understand than this.

Reply:
Yeah! we are checking that if theme function is not empty

Reply:
That function is a more formal way of checking the same thing precisely

Reply:
Easy to read code is preferable.

Reply:
shall i replace that by theme !== {}?

Reply:
@hetanthakkar1 I see that I was wrong on this one. Can you change it back and just add a comment explaining the code?

Reply:
@flyingcircle  done!